cmd/spark-connect-example-spark-session/main.go

```diff
@@ -19,16 +19,15 @@ package main
 import (
 	"context"
 	"flag"
-	"github.com/apache/spark-connect-go/v35/spark/sql/session"
 	"log"
 
+	"github.com/apache/spark-connect-go/v35/spark/sql/session"
+
```

Comment:
nit: rm newline

Reply:
my hope was that gofumpt actually fixes all of this, apparently not.

Reply:
![image](https://github.com/apache/spark-connect-go/assets/3421/c4215397-be35-42ba-b319-b81005cc02d7)

Actually it does